payments-core/src/main/java/com/stripe/android/payments/core/authentication/VoucherAuthenticator.kt

```diff
@@ -25,11 +25,10 @@ internal class KonbiniAuthenticator @Inject constructor(
         authenticatable: StripeIntent,
         requestOptions: ApiRequest.Options
     ) {
-        val detailsData = authenticatable.nextActionData as NextActionData.DisplayKonbiniDetails
+        val detailsData = authenticatable.nextActionData as NextActionData.DisplayVoucherDetails
         if (detailsData.hostedVoucherUrl == null) {
             ErrorReporter.createFallbackInstance(context).report(
-                ErrorReporter.UnexpectedErrorEvent.MISSING_HOSTED_VOUCHER_URL,
-                additionalNonPiiParams = mapOf("lpm" to "konbini")
```

Comment:
We regrettably lose a bit of type information in this error (and in the other LPM-specific authenticators), but since this is bound to a map via some Dagger magic, I couldn't think of any way to get the type information back. The tradeoff feels worth it, but open to other opinions!